.github/ISSUE_TEMPLATE/feature_request.md

```
@@ -0,0 +1,44 @@
+### Acceptance Criteria
+
+Include specific criteria for how to prove that this issue will be done.
+Ensure that each criterion is **measurable** and **testable** to clearly define what constitutes completion.
+
+- [ ] Task 1
+- [ ] Task 2
+- [ ] Task 3
+
+### Labels and Milestones
+
+**Remember to use helpful labels and milestones to categorize and track the progress of this feature request.**
```

Markdownlint (0.35.0)

Expected: 100; Actual: 111
Line length

(MD013, line-length)